src/wallet/wallet.cpp

```diff
@@ -56,6 +56,10 @@ CFeeRate CWallet::minTxFee = CFeeRate(DEFAULT_TRANSACTION_MINFEE);
  * Override with -fallbackfee
  */
 CFeeRate CWallet::fallbackFee = CFeeRate(DEFAULT_FALLBACK_FEE);
+/*
+ * Signal when transactions are added to wallet
+ */
+boost::signals2::signal<void (const CTransactionRef &ptxn, const uint256 &blockHash)> CWallet::TransactionAddedToWallet;
```

Comment:
Probably `CWalletTx` should be passed instead of `CTransactionRef`?

Reply:
actually if CWalletTX is used the blockhash is not needed anymore. I will take a look.